pkg/services/app/commit.go

```diff
@@ -0,0 +1,75 @@
+package app
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/fluxcd/go-git-providers/github"
+	"github.com/fluxcd/go-git-providers/gitprovider"
+	"github.com/weaveworks/weave-gitops/pkg/gitproviders"
+	"github.com/weaveworks/weave-gitops/pkg/utils"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+type CommitParams struct {
+	Name             string
+	Namespace        string
+	GitProviderToken string
+}
+
+func (a *App) GetCommits(params CommitParams) ([]gitprovider.Commit, error) {
```

Comment:
Adding a godoc comment here will help others know what this function does - since it is exported.